IOMC/EventVertexGenerators/src/BeamProfile2DB.cc

```diff
@@ -117,9 +103,9 @@ void BeamProfile2DB::fillDescriptions(edm::ConfigurationDescriptions& descriptio
   desc.add<double>("SigmaZ")->setComment("in cm");
   desc.add<double>("BetaStar")->setComment("in cm");
   desc.add<double>("Emittance")->setComment("in cm");
-  desc.add<double>("Alpha")->setComment("in radians");
-  desc.add<double>("Phi")->setComment("in radians");
-  desc.add<double>("TimeOffset")->setComment("in ns");
+  desc.add<double>("Alpha", 0.0)->setComment("in radians");
+  desc.add<double>("Phi", 0.0)->setComment("in radians");
+  desc.add<double>("TimeOffset", 0.0)->setComment("in ns");
   descriptions.addDefault(desc);
```

Comment:
```suggestion
  descriptions.addWithDefaultLabel(desc);
```

Reply:
addressed in [2bc4a06](https://github.com/cms-sw/cmssw/pull/41918/commits/2bc4a066bdf232e38ca496a0d6be5b75c354654b)